client/java/transports-gcplineage/src/main/java/io/openlineage/client/transports/gcplineage/GcpLineageTransportConfig.java

```diff
@@ -19,9 +19,9 @@
 public class GcpLineageTransportConfig
     implements TransportConfig, MergeConfig<GcpLineageTransportConfig> {
 
-  enum Mode {
-    sync,
-    async
+  public enum Mode {
```

Comment:
@ngorchakova Is there any test verifying this loaded correctly? The docs from here: https://openlineage.io/docs/client/java/configuration#configuration-6
suggests this should be lowercase.

Reading the config entry string to enum would require:
`objectMapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);`
feature to be turned on, as per https://stackoverflow.com/questions/24157817/jackson-databind-enum-case-insensitive
I am not sure if we have it on.